Dockerfile

```diff
@@ -28,6 +28,10 @@ RUN apt-get update -qq && \
 # http://unix.stackexchange.com/questions/195975/cannot-force-remove-directory-in-docker-build
 #        && rm -rf /var/lib/apt/lists
 
+# adding custom locales to provide backward support with scrapy cloud 1.0
+RUN echo 'en_US.UTF-8 UTF-8\nde_DE.UTF-8 UTF-8\nnl_NL.UTF-8 UTF-8\nfr_FR.UTF-8 UTF-8' \
+    >> /etc/locale.gen && locale-gen
```

Comment:
I think the important fact here is that we are using *official python images* for the stack aimed to be the default. It doesn't really matter if it is debian/ubuntu/centos/alpine as long as docker community thinks it is good as base image. We get security updates and well maintained image focused on python for free and I am sure docker community will communicate clearly when underlying distro is changed. Lot of users will have the same problem. What is more likely is that the underlying distribution is upgraded to debian 9.x (current is 8.4) and which is basically the same problem.

It is true the old and legacy stack, aka hworker, uses Ubuntu 12.04 but it already shows signs of obsolete and outdated system libraries. So in any case if we opt to use Ubuntu for scrapy stack it will be on a more recent release like 16.04 and even in that case we don't get the lastest Python 2 (or 3) version.

To me, at the moment there is only 1 distribution and it is the official python image :)